tools/conformance/djxl_conformance.cc

```diff
@@ -401,7 +400,6 @@ bool DecodeJXL(const DecodeOptions& opts) {
         std::vector<char> name(extra_channels[i].name_length + 1);
         EXPECT_SUCCESS(JxlDecoderGetExtraChannelName(dec.get(), i, name.data(),
                                                      name.size()));
```

Comment:
Doesn’t it mean that this is also not used? (Should it be?)

Reply:
I do suspect it *should* be, yes... I'm not really sure though